skiplang/prelude/ts/src/sk_worker.ts

```diff
@@ -201,6 +201,7 @@ export class PromiseWorker {
     this.source = ++sourcesLastId;
     this.callbacks = new Map();
     this.subscriptions = new Map();
+    // eslint-disable-next-line @typescript-eslint/no-this-alias
```

Comment:
It's here because later in the function uses to build the Promise this does not refers the the good object, but it's it's true that the function can probably be binded to this.